src/main/kotlin/systems/ajax/englishstudytelegrambot/entity/Entity.kt

```diff
@@ -1,12 +1,23 @@
 package systems.ajax.englishstudytelegrambot.entity
 
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+
+@Document(value = "users")
+data class User(@Id val telegramId: String, val username: String, val libraries: List<Library>)
```

Comment:
place each parameter on a new line. With such an approach, it can be easier to add new parameters to the class. fix in all entities